### DISCUSSION TITLE
AdNauseam and the upcoming extension limitations from google.

### DISCUSSION THREAD

**Discussion by Mistborn-First-Era** [dhowe/AdNauseam#2182] [Q&A] [answered]
Will AdNauseam still be working in chrome after google chrome goes to v3. Explain like I am five if you go in depth I have no clue what I am talking about.

**Comment by mneunomne** [comment]
Hey @Mistborn-First-Era, thank you for your question. we currently can't give you a definite answer to that, we are looking up the efforts from uBlock Origin to have a version for the v3 manifest, and see how AdNauseam can adapt to that. But *in theory*, yes AdNauseam should be adaptable for the v3 manifest version. I'll send a link to the discussion we will create regarding this.

**Comment by mneunomne** [comment, marked as answer]
see https://github.com/dhowe/AdNauseam/discussions/2188
